lnbits/core/models.py

```diff
@@ -141,6 +145,13 @@ class UpdateUserPassword(BaseModel):
     password: str = Query(default=..., min_length=8, max_length=50)
     password_repeat: str = Query(default=..., min_length=8, max_length=50)
     password_old: Optional[str] = Query(default=None, min_length=8, max_length=50)
+    username: Optional[str] = Query(default=..., min_length=2, max_length=20)
+
+
+class UpdateSuperuserPassword(BaseModel):
+    username: Optional[str] = Query(default=..., min_length=2, max_length=20)
```

Comment:
should `username` this be optional?

Reply:
fixed here: https://github.com/lnbits/lnbits/pull/1977/commits/8a25f65e8f62543a58190a59923c929e3145520b